### ISSUE TITLE
Repeated Bug when pressing Back

### ISSUE THREAD

**Issue by Havock94** [DexterHuang/CyberCodeOnline#2463]
I frequently encounter this bug when going to previous screen clicking "Back".
I'm noticing it on MacOS 12.2 with Chrome 100.0.4896.127

`PnR1cm59Zi5zaWJsaW5nLnJldHVybj1mLnJldHVybixmPWYuc2libGluZ31pPyhsPXIscz1vLnN0YXRlTm9kZSw4PT09bC5ub2RlVHlwZT9sLnBhcmVudE5vZGUucmVtb3ZlQ2hpbGQocyk6bC5yZW1vdmVDaGlsZChzKSk6ci5yZW1vdmVDaGlsZChvLnN0YXRlTm9kZSl9ZWxzZSBpZig0PT09by50YWcpe2lmKG51bGwhPT1vLmNoaWxkKXtyPW8uc3RhdGVOb2RlLmNvbnRhaW5lckluZm8saT0hMCxvLmNoaWxkLnJldHVybj1vLG89by5jaGlsZDtjb250aW4KICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIF4KCgpOb3RGb3VuZEVycm9yOiBGYWlsZWQgdG8gZXhlY3V0ZSAncmVtb3ZlQ2hpbGQnIG9uICdOb2RlJzogVGhlIG5vZGUgdG8gYmUgcmVtb3ZlZCBpcyBub3QgYSBjaGlsZCBvZiB0aGlzIG5vZGUuCkVycm9yOiBGYWlsZWQgdG8gZXhlY3V0ZSAncmVtb3ZlQ2hpbGQnIG9uICdOb2RlJzogVGhlIG5vZGUgdG8gYmUgcmVtb3ZlZCBpcyBub3QgYSBjaGlsZCBvZiB0aGlzIG5vZGUuCiAgICBhdCBwdSAoaHR0cHM6Ly9jeWJlcmNvZGVvbmxpbmUuY29tL3N0YXRpYy9qcy9tYWluLmFhNTI1NDMwLmNodW5rLmpzOjE6MjI2OTMwKQogICAgYXQgaHR0cHM6Ly9jeWJlcmNvZGVvbmxpbmUuY29tL3N0YXRpYy9qcy9tYWluLmFhNTI1NDMwLmNodW5rLmpzOjE6MjQwMzQxCiAgICBhdCB0LnVuc3RhYmxlX3J1bldpdGhQcmlvcml0eSAoaHR0cHM6Ly9jeWJlcmNvZGVvbmxpbmUuY29tL3N0YXRpYy9qcy9tYWluLmFhNTI1NDMwLmNodW5rLmpzOjE6MjYxMDAzKQogICAgYXQgV2kgKGh0dHBzOi8vY3liZXJjb2Rlb25saW5lLmNvbS9zdGF0aWMvanMvbWFpbi5hYTUyNTQzMC5jaHVuay5qczoxOjE4NTA0NCkKICAgIGF0IEVsIChodHRwczovL2N5YmVyY29kZW9ubGluZS5jb20vc3RhdGljL2pzL21haW4uYWE1MjU0MzAuY2h1bmsuanM6MToyMzgyMjApCiAgICBhdCBzbCAoaHR0cHM6Ly9jeWJlcmNvZGVvbmxpbmUuY29tL3N0YXRpYy9qcy9tYWluLmFhNTI1NDMwLmNodW5rLmpzOjE6MjM0MzQ0KQogICAgYXQgaHR0cHM6Ly9jeWJlcmNvZGVvbmxpbmUuY29tL3N0YXRpYy9qcy9tYWluLmFhNTI1NDMwLmNodW5rLmpzOjE6MTg1MzM0CiAgICBhdCB0LnVuc3RhYmxlX3J1bldpdGhQcmlvcml0eSAoaHR0cHM6Ly9jeWJlcmNvZGVvbmxpbmUuY29tL3N0YXRpYy9qcy9tYWluLmFhNTI1NDMwLmNodW5rLmpzOjE6MjYxMDAzKQogICAgYXQgV2kgKGh0dHBzOi8vY3liZXJjb2Rlb25saW5lLmNvbS9zdGF0aWMvanMvbWFpbi5hYTUyNTQzMC5jaHVuay5qczoxOjE4NTA0NCkKICAgIGF0IFFpIChodHRwczovL2N5YmVyY29kZW9ubGluZS5jb20vc3RhdGljL2pzL21haW4uYWE1MjU0MzAuY2h1bmsuanM6MToxODUyODAp`

**Comment by oashi** [comment]
I use the Back too, but did not experience that.
(Android 12 58.2 on Xperia 5 II)

**Comment by Havock94** [comment]
> I use the Back too, but did not experience that. (Android 12 58.2 on Xperia 5 II)

Yes the mobile app doesn't show the bug, it only occurred on desktop with a browser in my case
